test/WebSites/RazorPagesWebSite/RazorPagesWebSite.csproj

```diff
@@ -7,11 +7,17 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <EmbeddedResource Include="EmbeddedPages\**" />
+  </ItemGroup>
```

Comment:
Shouldn't these be embedded into the other library? If not then what's the point of it